R/methods-VariableMetadata.R

```diff
@@ -1,3 +1,27 @@
+# TODO tests for merge fxn
+
+#' Merge Variable*List objects
+#' 
+#' This function takes two VariableMetadataList or VariableSpecList objects and returns one.
+#' @param x Variable*List object
+#' @param y Variable*List object
+#' @return Variable*List object containing the entries from both inputs
+#' @export 
+setGeneric("merge",
+  function(x, y) standardGeneric("merge"),
+  signature = c("x", "y")
+)
+
+#'@export 
+setMethod("merge", signature("VariableMetadataList", "VariableMetadataList"), function(x,y) {
+  veupathUtils::VariableMetadataList(S4Vectors::SimpleList(c(as.list(x), as.list(y))))
+})
+
+#'@export 
+setMethod("merge", signature("VariableSpecList", "VariableSpecList"), function(x,y) {
```

Comment:
Can you make a `VariableMetadataList` from two `VariableSpecList`s?

I can't see how you could.

Reply:
Lol no. It's an error